### PR TITLE
[DDW-677] Implement alternative update path for manual updates - Improvements

### PR DESCRIPTION
**Pull request by nikolaglumac**
This PR includes the following fixes and improvements:
- Improved handling of a case where app-version API call fails (in which case we assume there is no newer version available and show the "Having trouble connecting/syncing" notification)
- Fixed handling of app version checks on `Windows` platform (the original API call was trying to fetch info for `platforms.win32.version` and the API responds with `platforms.windows.version`) - this was a major issue as on Windows we would never get version info 😱
- Simplified and improved logic behind the `isNewAppVersionLoading` variable which included reusing of `Request.wasExecuted` and `Request.isExecuting` flags
- Improved `externalRequest` method and URLs config
- Improved acceptance tests for app-version-check
- Added logic which hides the "Manual Update Overlay" during Node shutdown sequence

---

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/messages/GGKFXSKC6)
- [ ] Test

---

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

File: source/renderer/app/containers/LoadingPage.js [hunk 1]
```diff
@@ -72,17 +74,26 @@ export default class LoadingPage extends Component<InjectedProps> {
...
+          currentAppVersion={version}
+          availableAppVersion={availableAppVersion}
           isConnected={isConnected}
           isSynced={isSynced}
           isNodeStopping={isNodeStopping}
           isNodeStopped={isNodeStopped}
           isNotEnoughDiskSpace={isNotEnoughDiskSpace}
+          isNewAppVersionAvailable={isNewAppVersionAvailable}
+          isNewAppVersionLoading={getLatestAppVersionRequest.isExecuting}
+          isNewAppVersionLoaded={
```
Comment (tomothespian) [inline source/renderer/app/containers/LoadingPage.js]: @nikolaglumac can you isolate logic in separated variable outside component parameter declaration.

File: source/renderer/app/api/nodes/types.js [hunk 2]
```diff
@@ -51,36 +51,40 @@ export type GetNetworkStatusResponse = {
...
-  URL: string,
-  version: string,
   SHA256: string,
+  signature: string,
+  version: string,
+  URL: string,
 };
 
 export type Platforms = {
   ['windows' | 'darwin' | 'linux']: Platform,
```
Comment (tomothespian) [inline source/renderer/app/api/nodes/types.js]: When platform is windows json will return win32. @nikolaglumac can you improve that please?

File: source/renderer/app/containers/LoadingPage.js (hunk 1 above)
Comment (nikolaglumac) [reply source/renderer/app/containers/LoadingPage.js]: @tomothespian good point! Fixed now 🚀

**Comment by alan-mcnicholas** [comment]
I checked this on mainnet, staging and testnet and it looks good! :)

**Comment by nikolaglumac** [comment]
Version 0.13.2 removed from the `CHANGELOG` and `package.jsonà in: https://github.com/input-output-hk/daedalus/pull/1441

**Pull request merged**